real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/HillCrestImuNode.java

```diff
@@ -30,14 +40,85 @@ public HillCrestImuNode() {
 		Discovery discover = Discovery.getInstance();
 		discover.addListener(this);
 		
+		
 		// TODO Auto-generated constructor stub
 	}
 
+	
+	
 	@Override
 	public void handleRead(Device arg0, HidInMsg arg1, int arg2, long arg3) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The method handleRead() has an NPath complexity of 256](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2348033619/issues/source?bid=2938665&fileBranchId=3216826#l50)